pydantic/json_schema.py

```diff
@@ -372,7 +380,7 @@ def tuple_positional_schema(self, schema: core_schema.TuplePositionalSchema) ->
     def tuple_variable_schema(self, schema: core_schema.TupleVariableSchema) -> JsonSchemaValue:
         json_schema: JsonSchemaValue = {'type': 'array', 'items': {}}
         if 'items_schema' in schema:
-            json_schema['items'] = [self.generate_inner(s) for s in schema['items_schema']]
+            json_schema['items'] = [self.generate_inner(schema['items_schema'])]
```

Comment:
should items really be a list? looks to me like this suggests the array should only have one element?